be/src/runtime/routine_load/data_consumer.h

```diff
@@ -35,7 +36,8 @@ class DataConsumer {
     DataConsumer(StreamLoadContext* ctx):
         _init(false),
         _finished(false),
-        _cancelled(false) {
+        _cancelled(false),
+        _last_visit_time(0) {
```

Comment:
The initial last_visit_time is 0 ? Will it be clean in the beginning?

Reply:
No. the last_visit_time will be set when calling consumer's start(). And the clean thread can not see any active consumer, because active consumers are not in the pool.